Setup Prometheus /metrics endpoint for kepler model server for weights

KaiyiLiu1234 opened the pull request:
Setup a rough Prometheus /metrics endpoint for exporting weights. In the future, would like to consider using a lightweight sql database to improve runtime.

rootfs commented on server/kepler_model_trainer.py:
```diff
@@ -6,9 +6,10 @@
 
 # Dict to map model type with filepath
-model_type_to_filepath = {"core_model": "/models/core_model", "dram_model": "/models/dram_model"}
+model_type_to_filepath = {"core_model": "models/core_model", "dram_model": "models/dram_model"}
```
use absolute paths, e.g. `/var/lib/kepler-model-server`, this should also match the mount path [here](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/manifests/deployment.yaml#L71)

KaiyiLiu1234 replied:
I can do that, but I am also using os.path.dirname(__file__) and appending it to /models/core_model which should give the absolute path of the file it currently is in no matter where. Does that not work?

rootfs replied:
That should be fine, the path has to be deterministic so the deployment yaml and the source file can match.

KaiyiLiu1234 commented:
I changed the path back. This created a bug for os.path.join so I replaced that. I also added the desired fixes to the model and it works  better. I will likely reduce the epochs because I noticed that after a reduction in rmse, the rmse began to increase again.

rootfs commented on docker-compose.yaml:
```diff
@@ -0,0 +1,34 @@
+version: "3"
```
can you make a directory for this file and other files used by docker compose, e.g. under `deploy/docker-compose`?

KaiyiLiu1234 replied:
Ok will do. I will also update the read me.

rootfs replied:
cool, looking forward to the change
